discretisedfield/tests/test_mesh.py

```diff
@@ -551,6 +547,29 @@ def test_allclose(p1_1, p1_2, p2, n1, n2):
         mesh1.allclose(mesh3, rtol="1")
 
 
+def test_allclose_cell_accuracy():
```

Comment:
@fangohr This test outlines potential problems with the default `region.tolerance_factor` in `mesh.allclose`. However, it is probably not practically relevant.